ui/components/app/modals/contract-details-modal/index.scss

```diff
@@ -0,0 +1,99 @@
+.contract-details-modal {
+  background-color: var(--color-background-default);
+  width: 360px !important;
+
+  &__content {
+    padding: 24px 16px 32px 16px;
+    border-bottom: 1px solid var(--color-border-muted);
+
+    &__header {
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 24px;
+      display: flex;
+      align-items: center;
+    }
+
+    &__description {
+      font-style: normal;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 20px;
+      display: flex;
+      align-items: center;
+      letter-spacing: 0.25px;
+      padding-top: 8px;
+      color: var(--color-text-alternative);
+    }
```

Comment:
Most, if not all of this, can be replaced with the Typography component. line-height/letter-spacing may have to remain cc @georgewrmarshall might wanna take a look at supporting this. Honestly though line-height is part of our typography styles that are baseline, not prop driven, and probably shoudln't be overwritten unless for a good reason. Also not sure about the letter spacing at first glance.

Reply:
@brad-decker Done [here](https://github.com/MetaMask/metamask-extension/pull/15549/commits/d1ae5af192efb5c81d7eefe7c028e6a1223c6e37)